plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1459,6 +1465,8 @@ blockly.block.feature_tree_simple=Tree of type %1 with dirt: %2 trunk: %3 foliag
 blockly.block.feature_tree_simple.empty=(None)
 blockly.block.feature_tree_spruce=Spruce tree with dirt: %1 trunk: %2 foliage: %3 max foliage radius: %4 exposed trunk blocks: %5 roots: %6 height of %7 + at most %8 + at most %9 %10 always place dirt: %11 %12 ignore vines: %13 %14 decorators:
 blockly.block.feature_tree_spruce.empty=(None)
+blockly.block.feature_vegetation_patch=Vegetation patch on surface: %1 %2 horizontal radius: %3 vertical range: %4 %5 with chance %6 place vegetation feature: %7 ground block: %8 ground depth: %9 ground can replace blocks tagged in %10 %11 chance of extra ground depth: %12 %13 chance of extra ground column at the edge: %14
```

Comment:
![image](https://github.com/user-attachments/assets/b3a631b6-8f18-40b8-a371-e4fa2318729d)

you could probably omit the word "surface" here, so it goes "Vegetation patch on floor/ceiling"

Same for the waterlogged version